src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/ParseRequestService.java

```diff
@@ -50,24 +52,24 @@ public void validateUserProfiles(List retrievedProfiles, UserRequest userRequest
                 && userRequest.getUserIds().size() != retrievedProfiles.size()) {
             List<String> userIdsRetrieved = new ArrayList<>();
 
-            if (userType.equals(UserType.CASEWORKER)) {
+            if (nameContains(UserType.CASEWORKER.toString()).test(userType))  {
```

Comment:
we shouldn't replace equal operation with contains operation..I think, we shouldn't apply any predicate function here as it is reducing the readability of the code as well.